src/pks/mpc/mpc_coupled_dualmedia_water.cc

```diff
@@ -0,0 +1,157 @@
+#include "Teuchos_XMLParameterListHelpers.hpp"
```

Comment:
Ok, is the only thing keeping this MPC from working is being updated to new state?  Or is it not working/still in progress AND not updated to new state?

Reply:
This PK is still work in progress. I can update it to new state and make it compile, but it still will not be functional (today or tomorrow). Or I can remove this file for now. Or leave it as is.